src/tiledimage.js

```diff
@@ -241,6 +241,7 @@ $.TiledImage = function( options ) {
          * @property {OpenSeadragon.Tile} context - The HTML canvas context being drawn into.
          * @property {OpenSeadragon.Tile} rendered - The HTML canvas context containing the tile imagery.
          * @property {?Object} userData - Arbitrary subscriber-defined object.
+         * @deprecated
```

Comment:
The event is now deprecated. But, if one still wants to have continuous update just in time before a tile is drawn, the system still supports it: the user can access main cache data directly, and modify the tile accordingly (though they need to know what drawer is used and what the data type (and how) is stored, the webgl renderer will not support such interaction.

So while I don't want to have two events per a tile draw, too many unnecessary calls, I would like to have one event allowing for sync modification. `tile-drawn` could be a nice replacement, but, it is called _after_ the event which might be useless. What about moving that event before the draw call?

Reply:
I will wait for the drawer API change merge, then it will be more obvious..

Reply:
To me the phrase "tile drawn" implies that it has already drawn, so I don't think it should keep the same name if it happens before the tile is drawn.